src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\Crypt32\Interop.certificates_types.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\Crypt32\Interop.certificates_imports.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.FormatMessage.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.GetModuleHandle.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\winhttp\Interop.SafeInternetHandle.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\winhttp\Interop.winhttp_types.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\winhttp\Interop.winhttp_imports.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)\System\Net\Http\CertificateHelper.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)\System\Net\Http\\WinHttpChannelBinding.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)\System\Net\Http\\WinHttpException.cs" />
```

Comment:
@bartonjs Thanks for pointing that out.  The double back-slashiness was caused by search/replace macro error.